src/Xamarin.Android.Build.Tasks/Tests/Xamarin.ProjectTools/Xamarin.ProjectTools.csproj

```diff
@@ -91,6 +91,9 @@
     <Compile Include="Common\DotNetXamarinProject.cs" />
     <Compile Include="XABuildPaths.cs" />
   </ItemGroup>
+  <ItemGroup>
+    <AndroidEnvironment Include="Environment.txt" />
```

Comment:
I dont think this is needed. It won't be used by anything since Xamarin.ProjectTools is not used on a device or in any app. Its purely a testing framework that we run within NUnit on the host machine.